src/bin/vip-logs.js

```diff
@@ -43,14 +48,47 @@ export async function getLogs( arg: string[], opt ): Promise<void> {
 
 	await trackEvent( 'logs_command_success', {
 		...trackingParams,
-		logs_output: logs.length,
+		total: logs.nodes.length,
 	} );
 
-	if ( ! logs.length ) {
+	if ( ! logs.nodes.length ) {
 		console.error( 'No logs found' );
 		return;
 	}
 
+	printLogs( logs.nodes, opt.format );
+}
+
+export async function followLogs( opt ): Promise<void> {
+	let after = null;
+	let isFirstRequest = true;
+
+	while ( true ) {
+		const limit = isFirstRequest ? opt.limit : 5000;
+
+		let logs;
+		try {
+			logs = await logsLib.getRecentLogs( opt.app.id, opt.env.id, opt.type, limit, after );
+		} catch ( error ) {
+			console.error( 'Failed to fetch logs. Trying again after the polling interval' );
```

Comment:
Do we want to have some sort of exponential backoff as well as a failure limit? If the API is down for an extended period (or if the computer has network issues), it would be better to have any following clients ease off and automatically stop rather than polling forever.

Reply:
Done! I've refactored the delay handling, so it informs the user when it's going to retry, we have a backoff (not exponential, but we don't want to wait too much in case of a sequence of errors, I think). I've also changed the API call method so it doesn't immediately exit the process for the logs command, as it was preventing retries on certain errors (such a network error)